vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -396,8 +396,7 @@ function Test-RegistryAdded {
         Validate-WindowsFixInFeatureManagement -Name 747051149
     }
     if ($env:WindowsSKU -Like '23H2*') {
-        Validate-WindowsFixInHnsState -Name NamespaceExcludedUdpPorts -Value 65330
-        Validate-WindowsFixInHnsState -Name PortExclusionChange -Value 1
+        Validate-WindowsFixInHnsState -Name PortExclusionChange -Value 0
```

Comment:
Do you also need https://github.com/Azure/AgentBaker/commit/f5a504eb139c1496b6eee3911c6b3771c6a6c2c9?

Reply:
Already cherry pick this one.